src/components/TimeTravel/TimeTravel.js

```diff
@@ -351,9 +363,13 @@ TimeTravel.propTypes = {
    */
   onTimestampInputEdit: PropTypes.func,
   /**
-   * Optional callback handling clicks on timeline timestamp labels (e.g. for tracking)
+   * Optional callback handling clicks on timeline pan buttons (e.g. for tracking)
+   */
+  onTimelinePanButtonClick: PropTypes.func,
```

Comment:
This is never called anywhere.

Reply:
Good catch, I will modify the `handleTimelinePanButtonClick` function!